.github/workflows/ci-cd.yaml

```
@@ -262,6 +264,8 @@ jobs:
           echo "DJANGO_OPEN_AI_SECRET_KEY=${{ secrets.DJANGO_OPEN_AI_SECRET_KEY }}" >> .env.backend
           echo "DJANGO_SECRET_KEY=${{ secrets.DJANGO_SECRET_KEY }}" >> .env.backend
           echo "DJANGO_SENTRY_DSN=${{ secrets.DJANGO_SENTRY_DSN }}" >> .env.backend
+          echo "DJANGO_SENTRY_RELEASE=$(echo ${{ github.sha }} | cut -c1-7)" >> .env.backend
```

Let's use a more common name
```suggestion
          echo "DJANGO_RELEASE_VERSION=$(echo ${{ github.sha }} | cut -c1-7)" >> .env.backend
```